newIDE/app/src/UI/EditorMosaic/index.js

```diff
@@ -117,6 +119,17 @@ const renderMosaicWindowPreview = props => (
   </div>
 );
 
+const renderMosaicWindowToolbar = props => (
+  <div className="mosaic-window-toolbar" style={{ width: '100%' }}>
```

Comment:
Can you confirm that this is the implementation found in react-mosaic? :)

Reply:
(want to be sure we're not forgetting anything)

Reply:
I guess we are maybe adding just one extra div, (and I need it to wrap other divs, using React.Fragment throws an error). Though this is the implementation:
https://github.com/nomcopter/react-mosaic/blob/b36984c34b0fa1e3f21a211578301933cea24992/src/MosaicWindow.tsx#L136-L150
I don't really see any other difference, (especially in devTools, there's just one more div, which is used for wrapping) Is it fine, what do you suggest?

Reply:
What's the error with React.Fragment?

Reply:
Error: Only native element nodes can now be passed to React DnD connectors.You can either wrap the component into a <div>, or turn it into a drag source or a drop target itself.

Reply:
Ok, last questions:

* You said "there's just one more div, which is used for wrapping) Is it fine, what do you suggest?"
=>  what is this extra div? 

* Can you add a comment with a link to MosaicWindow.tsx telling "this is based on the implementation of the react-mosaic window toolbar done here: [link]"

Thanks :)

Reply:
https://github.com/nomcopter/react-mosaic/blob/b36984c34b0fa1e3f21a211578301933cea24992/src/MosaicWindow.tsx#L145-L149 
You can see here that our custom toolbar is injected inside a `<div className="mosaic-window-toolbar">`.

Now, inside this div, there should be two divs, one for title and one for toolbar controllers. They also have to be siblings of each other. So to maintain their sibling relationship in the DOM tree I'm wrapping them inside a `<div className="mosaic-window-toolbar">` again. This is the extra div I'm talking about (which is not present in the actual implementation). So, the DOM tree would look like this,

`<div class="mosaic-window-toolbar">` => this is the parent div
`.     <div class="mosaic-window-toolbar">` => this is the additional wrapping div
`. .            <div class="mosaic-window-title"></div>` => this div is for the title
`. .            <div class="mosaic-window-controls"></div>` => this is for controls
`.      </div>`
`</div>`

Now visually the additional wrapping div overlays the entire area of the actual parent div since they have the similar styling and dimensions. Ideally, React.Fragment should've been used for wrapping instead of div, but it is not a react-dnd element. 

> Can you add a comment with a link to MosaicWindow.tsx telling "this is based on the implementation of the react-mosaic window toolbar done here: [link]"

Sure thing :)

Reply:
My spidey sense is clearly afraid of making a duplicate `mosaic-window-toolbar`. There is 9 chance out of 10 that this will create some weird issue in the future (someone create a theme, customize the CSS to add a border => why the heck is the border applied twice??... these kind of things that easily takes hours to understand and fix).

So much added complexity just for a tooltip 😬 I'm almost tempted to back off this implementation and just pass title as a string (using i18n._(t`Blabla`)).

With this no more re-definition of things that should be handled by react-mosaic. It should almost be an improved to react-mosaic to be able to pass a different tooltip... even if the tooltip itself is not the most important stuff we need to be honest.

Sorry for noticing only now.. :/ Clearly not satifsying solution here. Let's consider if we can just use i18n._, refactor the title to be using the `t` string litteral, and call it a day?